env.py

```diff
@@ -11,6 +11,8 @@
 from gym_minigrid.minigrid import Grid
 import numpy as np
 
+from custom_envs import *
```

Comment:
I think best practice is to import exactly what you need and never import *. just a minor note, not critical.